.github/BOTMETA.yml

```diff
@@ -987,8 +987,9 @@ files:
   $modules/remote_management/oneview/oneview_fcoe_network.py:
     authors: fgbulsoni
   $modules/remote_management/redfish/:
-    authors: jose-delarosa
+    authors: jose-delarosa billdodd
```

Comment:
I'm not really sure why this line is needed at all, since the author information is usually taken from the modules themselves. @gundalow do you know why we list authors here as well?